cells/_automation/nixago.nix

```diff
@@ -69,4 +69,19 @@ in {
       ];
     };
   };
+  std-gh-action = presets.nixago.std-gh-action {
+    configData = {
+      on.push.branches = ["main"];
+      targets = [
+        {
+          block = "cli";
+          action = "build";
+        }
+        {
+          block = "packages";
+          action = "build";
+        }
```

Comment:
Unfortunately, I don't see a way how we could scrape this from `__std.ci`. Any ideas?